server/settings/settings.proto

```diff
@@ -3,13 +3,13 @@ option go_package = "github.com/argoproj/argo-cd/server/settings";
 
 // Settings Service
 //
-// Settings Service API retrives ArgoCD settings 
+// Settings Service API retrives Argo CD settings
```

Comment:
Please apply the same change in https://github.com/argoproj/argo-cd/blob/master/server/settings/settings.pb.go#L9